src/js/pages/Modal.js

```diff
@@ -0,0 +1,68 @@
+import Component from "../lib/Component.js";
+import {$} from "../modules/utils.js";
+
+export default class Modal extends Component {
+  template() {
+    const {isModalVisible, winnings} = this.props;
```

Comment:
```suggestion
    const {isModalVisible, winnings: {rankings, profitRate} = this.props;
```
한번 더 분해하면 아래 반복적인 프로퍼티 접근을 줄일 수 있을꺼 같아요:)